src/views/docs-view/server.ts

```diff
@@ -168,6 +168,22 @@ export function getStaticGenerationFunctions<
 				pathParts.join('/')
 			)}`
 			const headings: AnchorLinksPluginHeading[] = [] // populated by anchorLinks plugin below
+			// Hide the version selector on TFE release notes page, e.g. /terraform/enterprise/<year>/<version> or /vault/docs/release-notes/1.13.0
+			const isReleaseNotesPage = (path) => {
+				return /(\/releases\/[0-9]{4}\/(v[0-9]{6}-\d+))/i.test(path) ||
+					/\/release-notes\/\d+[.]\d+[.](\d+|x)/i.test(path)
+					? true
+					: false
+			}
```

Comment:
Would you mind moving this to either an exported function in this file or to somewhere like `lib` so we can add tests in the future? (I don't think we need to add tests at the moment, but given that it's going to run for every URL I'd like to eventually have them for reassurance!)

Reply:
You got it!

Reply:
done 8faa8ec4a0392f70f524fdb68391e932978c6374